### ISSUE TITLE
Return the current version of the extension

### ISSUE THREAD

**Issue by seachicken** [cli/go-gh#25]
I would like to see which version the extension users are using in order to get feedback on the extension.
How about creating a helper function here that returns the version in the `tag` in the `manifest.yml` that is created when a binary extension is installed?

**Comment by samcoe** [comment]
@seachicken I am not sure I understand your feature request. Can you give me a bit more detail on what goal you are trying to achieve? 

From my current understanding it seems like a `gh` extensions that is trying to get feedback about the currently installed version would be able to do so without the help of `go-gh` by simply specifying a version in the extension itself that gets updated when a new release is created.

**Comment by samcoe** [comment]
@seachicken I understand now, thank for explaining. I think right now writing a `gh poi --version` is the right solution, but you are correct that there is no place that we list extension versions in `gh`. It seems like this is information that should be output in `gh extension list`, if you agree I will move this issue over to the `cli/cli` repo where that command is maintained and we can get this feature added.

**Comment by seachicken** [comment]
Yes, it would be nice to be able to check the version in the `gh extension list` 😃

**Comment by seachicken** [comment]
However, one concern with showing the version in the `gh extension list` is that people who are currently creating extensions with sh don't have a `manifest.yml` and seem to be comparing them with sha. https://github.com/cli/cli/blob/c3d451b9b6818a01b66dec9fb82ec1a91fbfd280/pkg/cmd/extension/manager.go#L231
When we run the `gh extension list`, the sh extensions will not show the version, so it may not look consistent.
I'm not sure if it would be easy to add a `manifest.yml` to the sh extension as well.
